src/controllers/itemdetailpage.js

```diff
@@ -982,6 +993,10 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
             if (overview) {
                 elem.innerHTML = overview;
                 elem.classList.remove("hide");
+
+                elem.classList.add("detail-clamp-text");
+                elem.addEventListener('click', toggleLineClamp);
```

Comment:
It does not work on TV, as the element is not focusable.

Ideally, there should be a "Show more" link-button that unveils the element.